plugins/rtmp-services/data/services.json

```diff
@@ -1,5 +1,5 @@
 {
-    "format_version": 1,
+    "format_version": 2,
```

Comment:
Well they renamed the service entry so it's probably not necessary to increment the format version anymore.